specs-go/config.go

```diff
@@ -16,8 +16,9 @@ type Spec struct {
 
 // Device is a "Device" a container runtime can add to a container
 type Device struct {
-	Name           string         `json:"name"`
-	ContainerEdits ContainerEdits `json:"containerEdits"`
+	Name           string            `json:"name"`
+	Annotations    map[string]string `json:"annotations,omitempty"`
```

Comment:
I know we haven't really done it up to now, but we could add a docstring for the `Annotations` to explain their intent.

Reply:
Added the docstring, where do you want to have that short description?

Reply:
Sorry, missed this one. I think the readme where we show the example we could expand on this. I don't think it's a blocker for the PR though.